package.json

```diff
@@ -5,18 +5,22 @@
 	"author": "ISV SWAT",
 	"bugs": "https://github.com/forcedotcom/sfdx-scanner/issues",
 	"dependencies": {
-		"@lwc/eslint-plugin-lwc": "^0.10.0",
+		"@babel/core": "^7.11.0",
+		"@babel/eslint-parser": "^7",
+		"@lwc/eslint-plugin-lwc": "~1.1.2",
```

Comment:
Would we be restricting the users to a specific version and cause them to miss out on the latest minor versions? For PMD, we've had to release everytime there's a new minor version, but since eslint has been an npm dependency, we didn't have to actively keep up.

According to [NPM's documentation](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#dependencies), `^` would make sure the versions would be compatible.

Based on this, my vote would be keep it as
```
"@lwc/eslint-plugin-lwc": "^1.1.2"
```